### DISCUSSION TITLE
佩佩溜冰冰

### DISCUSSION THREAD

**Discussion by Mapleawaa** [share121/inter-knot#1700] [其他]
# 她真可爱
![1723227367036](https://github.com/user-attachments/assets/ff29e57c-2b1a-4bbe-abee-65c0d1ac1f5f)
### 她真可爱
![1723227371112](https://github.com/user-attachments/assets/0a8019be-aeb4-44f9-a7d0-da9a382c711b)
#### 她真可爱
 - [x] 她真可爱
粥吧打过来了 x

**Comment by lllliufeng** [comment]
She is so __________.
A. cute
B. adorable
C. moe
D. obtuse angle

**Reply by BlessedinHighlight** [reply]
钝角

**Comment by jinuo12345** [comment]
确实

**Comment by ghost** [comment]
舟没有自己的论坛吗

**Reply by hanshui2077** [reply]
哈哈这本来就相当于是一个大众论坛嘛

**Reply by ghost** [reply]
正是舟都是这种想法所以舟到处殖民还出了一个叫殖民地的游戏

**Reply by zxyDEDIRE** [reply]
好家伙，对粥意见有点大啊你

**Reply by ghost** [reply]
> 好家伙，对粥意见有点大啊你

咋的不允许吗？利刃冰清玉洁？什么米家相关论坛都殖民

**Reply by ghost** [reply]
> > > 好家伙，对粥意见有点大啊你
> > 
> > 
> > 咋的不允许吗？利刃冰清玉洁？什么米家相关论坛都殖民
> 
> 唉，不顺眼就直接举报吧，owner处不处理另说，在这也争不出个什么来

好

**Reply by AnbulesPeo** [reply]
别这样吧，正常的讨论应该还是应该容许的，大家都和和气气的不要吵架不是更好吗

**Reply by Mapleawaa** [reply]
> > > > 好家伙，对粥意见有点大啊你
> > > 
> > > 
> > > 咋的不允许吗？利刃冰清玉洁？什么米家相关论坛都殖民
> > 
> > 
> > 唉，不顺眼就直接举报吧，owner处不处理另说，在这也争不出个什么来
> 
> 好

不是哥们 发粥怎么了 发米家之外的你就出警了？

**Reply by ghost** [reply]
> > > > > 好家伙，对粥意见有点大啊你
> > > > 
> > > > 
> > > > 咋的不允许吗？利刃冰清玉洁？什么米家相关论坛都殖民
> > > 
> > > 
> > > 唉，不顺眼就直接举报吧，owner处不处理另说，在这也争不出个什么来
> > 
> > 
> > 好
> 
> 不是哥们 发粥怎么了 发米家之外的你就出警了？

咋滴，粥就能不断殖民？如果米家之外不能出警，这个网站别叫绳网好吧，吃着绝区零的流量然后成为殖民的自留地是吧，披着米家的皮把人骗进来杀是吧

**Reply by ghost** [reply]
> > > > > 好家伙，对粥意见有点大啊你
> > > > 
> > > > 
> > > > 咋的不允许吗？利刃冰清玉洁？什么米家相关论坛都殖民
> > > 
> > > 
> > > 唉，不顺眼就直接举报吧，owner处不处理另说，在这也争不出个什么来
> > 
> > 
> > 好
> 
> 不是哥们 发粥怎么了 发米家之外的你就出警了？

发粥怎么了？粥就可以明目张胆的殖民？喜欢发粥自己做一个网站去，叫什么殖民地网，大英帝国，日不落帝国，印第安人的天堂谁管你啊，非要在一堆米家内容里发粥的，煞风景是吧，粥真的是夸张哦，噶别人头皮还理直气壮的

**Reply by ghost** [reply]
> > > > > 好家伙，对粥意见有点大啊你
> > > > 
> > > > 
> > > > 咋的不允许吗？利刃冰清玉洁？什么米家相关论坛都殖民
> > > 
> > > 
> > > 唉，不顺眼就直接举报吧，owner处不处理另说，在这也争不出个什么来
> > 
> > 
> > 好
> 
> 不是哥们 发粥怎么了 发米家之外的你就出警了？

幸亏今天不是感恩节，不然还要给英大人吃点火鸡

**Reply by bunsoutA** [reply]
别给米添黑了，就是你这种人才给米败坏路人缘。有什么事情不能好好说，非得到处阴阳人。

**Reply by bunsoutA** [reply]
这就是扩圈的坏处 ，别把什么样的都给扩进来了

**Reply by ghost** [reply]
> 别给米添黑了，就是你这种人才给米败坏路人缘。有什么事情不能好好说，非得处阴阳人。

特有歪屁股双畜讲路人缘，米需要个锤子路人缘，米的路人缘早在原神时期就被黑子说烂了，米圈现在比二游圈都大，咋的，粥圈算路人是吧，噶头皮还对英大人说我没有败坏印第安人的路人缘😭

**Comment by devil233-ui** [comment]
唉，不顺眼就直接举报吧，owner处不处理另说，在这也争不出个什么来

**Reply by Mapleawaa** [reply]
发生什么事了

**Reply by devil233-ui** [reply]
> 发生什么事了

如果你真的是双厨+萌萌人，上b站或知乎搜一下米粥之间的历史...~~虽然我感觉还在玩贴吧的不了解不太可能~~

**Reply by Mapleawaa** [reply]
> > 发生什么事了
> 
> 如果你真的是双厨+萌萌人，上b站或知乎搜一下米粥之间的历史...~虽然我感觉还在玩贴吧的不了解不太可能~

看着楼上反应这么激烈就知道了orz 入圈比较**晚**是真不知道 这下还真是 ~绳网特有的见多识广~

**Reply by devil233-ui** [reply]
反正如果真的在玩贴吧，还是建议尽早脱离为妙，唐人街（nga）也绝对不要去碰，远离蓝黄去哪都行

**Reply by Mapleawaa** [reply]
了解了，感谢

**Comment by cpu644** [comment]
特有的米粥哲学问题

**Reply by Mapleawaa** [reply]
是这样的说 两家玩家都有极端

**Comment by AKT505** [comment]
节奏打过来了？

**Comment by xioagong** [comment]
是她自己跑到岛上的
